mandatory/2-function-creation.js

```diff
@@ -25,7 +38,14 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  // let removeSpace = arrayOfStrings.trim();
+  // let replaceSlash = removeSpace.replace("/");
+  // let lowercase = replaceSlash.toLowerCase();
+  // using Map Function
+  let String = arrayOfStrings.map((x) => x.trim().replace("/").toLowerCase());
```

Comment:
When using the `replace` method, you should tell js what you want to replace with. Something like this
```
let String = arrayOfStrings.map((x) => x.trim().replace("/", "").toLowerCase());
```
Note the empty string in the `replace` method, this tell Js to replace `/` with nothing ``